Carte-jmeter.jmx

```diff
@@ -58,7 +58,7 @@
           <collectionProp name="Arguments.arguments">
             <elementProp name="demo_files_dir" elementType="Argument">
               <stringProp name="Argument.name">demo_files_dir</stringProp>
-              <stringProp name="Argument.value">d:\Work\Pentaho\Kettle.git\testfiles\jmeter</stringProp>
+              <stringProp name="Argument.value">d:/Work/Pentaho/Kettle.git/testfiles/jmeter</stringProp>
```

Comment:
I don't think we want a real file path to be in this property, especially one pointing to a dev sandbox.